main/meson/APKBUILD

```diff
@@ -2,13 +2,13 @@
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=meson
 pkgver=0.50.1
-pkgrel=0
+pkgrel=1
 pkgdesc="Fast and user friendly build system"
 url="https://mesonbuild.com"
 arch="noarch"
 license="Apache-2.0"
-depends="ninja python3"
-makedepends="python3-dev"
+depends="ninja py3-setuptools python3"
+makedepends="py3-setuptools"
```

Comment:
Why is `py3-setuptools` listed twice?

Please decide if it is needed anytime meson is installed or only while creating the meson package.